Sources/MapboxMaps/Foundation/MapView.swift

```diff
@@ -618,8 +620,27 @@ open class MapView: UIView {
         cameraAnimatorsRunnerEnablable.isEnabled = true
 
         updateDisplayLinkPreferredFramesPerSecond()
+
+        // this will make sure that display link is only running on an active scene in foreground,
+        // preventing metal view drawing on background if the view is added to window not on foreground
```

Comment:
TBH I did expect that this documentation part will become a method documentation 🙂